docs/getting-started/install-cent-node.md

```diff
@@ -69,52 +97,35 @@ Once you have installed all the necessary packages, follow these steps to instal
   -a 8082 -p 38204
   ```
 
-  This command automatically adds the Identity keys to the `config.yaml` file.
+  This command automatically creates an Identity and keys needed and generates the `config.yaml` file required to run the node.
   Replace the `_KEY-FILE_` with the key file you obtained when creating the Ethereum account and `_PATH-TO-CONFIG-FILE_` with the location where you want the `config.yaml`  file to be stored.
 
-3. Run the Centrifuge Node using the `config.yaml` file you created:
+4. Run the Centrifuge Node using the `config.yaml` file you created:
 
   ```bash
   $ centrifuge run -c /PATH-TO-CONFIG-FILE/config.yaml
   ```
   Replace the `_PATH-TO-CONFIG-FILE_` with the location of the `config.yaml` file.
 
-### Installing the Ethereum Rinkeby Node
+#### Additional custom configuration
+* Configure node under NAT
 
-Rinkeby is a testnet
+If you want your node to be reachable outside of your private network you will need to manually specify the External IP where the node will be available:
```

Comment:
If you want your node to be accessible outside your private network, you will need to manually specify the External IP of the node: